internal/osutils/user/user.go

```diff
@@ -0,0 +1,44 @@
+package user
+
+import (
+	"os"
+
+	"github.com/ActiveState/cli/internal/constants"
+)
+
+// HomeDirNotFoundError is an error that implements the ErrorLocalier and ErrorInput interfaces
+// from locale/errors.go because importing locale for NewInputError creates an import cycle.
+// Instead, return this error that looks like a LocalizedError.
+type HomeDirNotFoundError struct {
+}
+
+const homeDirNotFoundErrorMessage = "Could not proceed because your HOME environment variable is unset. " +
+	"Please ensure that your HOME environment variable is set. " +
+	"Alternatively if you do not or cannot set this variable you can instead use the ACTIVESTATE_HOME variable." +
+	"\n\n" +
+	"This variable is used by the State Tool to determine things like the installation directory, config directory and cache directory."
+
+func (e *HomeDirNotFoundError) Error() string {
+	return homeDirNotFoundErrorMessage
```

Comment:
Please consider having the error message be something simple and possibly overly informative (i.e. for developers only). It also needs to implement the Unwrap method: `func (e *HomeDirNotFound) Unwrap() error {` so that the dev messaging contains the cause.